src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -372,4 +378,13 @@ export default class AdvancedFilters extends Vue {
     await this.storeModule.updateFilterConfigLive({ ...this.filterElementValueLive, ...this._planeAdvancedData });
     this._emitAdvancedFiltersChange();
   }
+
+  private _expandCollapseAccordions(event: any) {
+    event.preventDefault();
+    if (this._advancedFiltersAccordion) {
+      this._isExpanded ? this._advancedFiltersAccordion.collapseAll() : this._advancedFiltersAccordion.expandAll();
+      this._isExpanded = !this._isExpanded;
+      this.$forceUpdate();
```

Comment:
This should not be necessary.

If your intention is to rerender the `<a>` element when toggling the text between Expand All and Collapse All, you could simply remove `_` from the name of the property
`this._isExpanded` --> `this.isExpanded`
and make its' change reactive.

Changing properties with a leading underscore is not reactive in Vue.